test/linter/test-versions.js

```diff
@@ -25,6 +25,14 @@ const VERSION_RANGE_BROWSERS = {
   webview_android: ['≤37'],
 };
 
+/** @type {Object<string, string>} */
+const browserTips = {
+  safari_ios:
+    'The version numbers for Safari for iOS are based upon the iOS version number rather than the Safari version number.  Maybe you are trying to use the desktop version number?',
+  opera_android:
+    'Blink editions of Opera Android and Opera desktop were the Chrome version number minus 13, up until Opera Android 43 when they began skipping Chrome versions.  Please double-check browsers/opera_android.json to make sure you are using the correct versions.',
+};
```

Comment:
Duplicate spaces, but maybe that's intentional?

```suggestion
const browserTips = {
  safari_ios:
    'The version numbers for Safari for iOS are based upon the iOS version number rather than the Safari version number.  Maybe you are trying to use the desktop version number?',
  opera_android:
    'Blink editions of Opera Android and Opera desktop were the Chrome version number minus 13, up until Opera Android 43 when they began skipping Chrome versions.  Please double-check browsers/opera_android.json to make sure you are using the correct versions.',
};
```